app/dashboard/tokens.py

```diff
@@ -200,3 +200,10 @@ def addr_to_token(addr):
         if(token['addr'].lower() == addr.lower()):
             return token
     return False
+
+
+def token_by_name(name):
+    for token in tokens:
```

Comment:
`tokens` is undefined in this scope.

Reply:
I pushed a fix as soon as it popped on rollbar the other day? 🤔 https://github.com/gitcoinco/web/commit/2325aa7aa867fc7df29f3088172fc4a00a4cb513

Reply:
thx; i think i must have fixed another 500 issue on /activity then